ammo-fe/cypress/integration/Error.cy.ts

```diff
@@ -6,15 +6,15 @@ describe('Error boundary behaviors', () => {
         cy.createBullet(bulletMock);
     });
 
-    it('Should redirect to home in case of a 404 - Bullets must be visible', () => {
+    it('Visiting unknow URL, being redirected on 404 page and going back to home page - bullets should still be visible', () => {
```

Comment:
'it visit' might be better than 'it visiting' ?

Reply:
Like we discussed, I'm not taking the 'it' into account because it won't appear in tests reports, coverage, console etc.